src/main/java/org/myrobotlab/codec/CodecUtils.java

```diff
@@ -532,4 +549,63 @@ public static void main(String[] args) {
     }
   }
 
+  /**
+   * Creates a properly double encoded Json msg string.

```

Comment:
feels icky to have to double encode/decode anything.

Reply:
to support "distributed" it necessary at the moment ...
but I didn't want to omni bus with pulling the header into a separate sub object

Reply:
won't fix - this pr